global-script.lua

```diff
@@ -3890,3 +3896,36 @@ function onPlayerDisconnect(player)
         updatePlaySpiritButton(player.color)
     end
 end
+
+function onObjectSpawn(obj)
+    if obj.hasTag("Minor") or obj.hasTag("Major") or obj.hasTag("Unique") then
+        applyPowerCardContextMenuItem(obj)
+    end
+end
+
+function applyPowerCardContextMenuItem(card)
+    card.addContextMenuItem(
+        "Forget",
+        function()
+            -- This ugliness is because setPositionSmooth doesn't work from a hand.
+            ensureCardInPlay(card)
+            discardPowerCardFromPlay(card, 1)
+        end,
+        false)
+end
+
+-- ensureCardInPlay moves the supplied card from a player's hand to a safe
+-- location, if it's in a hand.
+function ensureCardInPlay(card)
+    for playerColor, _ in pairs(selectedColors) do
```

Comment:
it's probably better to iterate through all hands, and not just hands of selected colors in case a player puts a card in a different hand. so instead of `selectedColors` something like `PlayerBags` could work

Reply:
Done.